sklearn/datasets/lfw.py

```diff
@@ -44,14 +44,15 @@
 logger = logging.getLogger(__name__)
 
 
-BASE_URL = "http://vis-www.cs.umass.edu/lfw/"
```

Comment:
hmm, I believe so. I specifically made an attempt to maintain backwards compatibility by changing the associated variable names